wgsl/index.bs

```diff
@@ -262,6 +262,7 @@ spec: WebGPU; urlPrefix: https://gpuweb.github.io/gpuweb/#
         text: front-facing; url: front-facing
         text: shader-output mask; url: shader-output-mask
         text: framebuffer; url: framebuffer
+        text: normalized device coordinate; url: ndc
```

Comment:
µnit: if bikeshed will allow it, pluralize this; definition is for the coordinates as a whole.